api/src/main/java/com/exelaration/abstractmemery/domains/AuditModel.java

```diff
@@ -0,0 +1,27 @@
+package com.exelaration.abstractmemery.domains;
```

Comment:
Should this file be removed? I took it out previously - I'm not sure how it got back in

Reply:
Idk how either, but nice catch I'll delete it